Hi3Helper.Core/Lang/en.json

```diff
@@ -589,7 +589,7 @@
 
   "_InstallMigrate": {
     "PageTitle": "Game Migration",
-    "Step1Title": "Process will start in 5 seconds. Please accept the UAC dialog to start Migration process",
+    "Step1Title": "Process will start in 5 seconds. Please accept the UAC dialog to start the migration process",
```

Comment:
I think we can remove ``Please accept the UAC dialog to start the migration process`` as the launcher is now running with administrator privilege by default and the requirement for access elevation is no longer necessary.

Reply:
Fixed.